yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -437,17 +437,17 @@ defaultForgotPasswordHandler = do
         |]
   where
     forgotPasswordForm extra = do
-         (emailRes, emailView) <- mreq emailField emailSettings Nothing
+        (emailRes, emailView) <- mreq emailField emailSettings Nothing
     
-         let forgotPasswordRes = ForgotPasswordForm <$> emailRes
-         let widget = do
-             [whamlet|
-                 #{extra}
-                 ^{fvLabel emailView}
-                 ^{fvInput emailView}
-             |]
-
-	 return (forgotPasswordRes, widget)
+        let forgotPasswordRes = ForgotPasswordForm <$> emailRes
+        let widget = do
+            [whamlet|
+                #{extra}
+                ^{fvLabel emailView}
+                ^{fvInput emailView}
+            |]
+        return (forgotPasswordRes, widget)
+
```

Comment:
there was a tab here, in case you're wondering what this seemingly redundant change is.